src/execution/physical_plan/plan_get.cpp

```diff
@@ -160,8 +160,24 @@ unique_ptr<PhysicalOperator> PhysicalPlanGenerator::CreatePlan(LogicalGet &op) {
 		vector<unique_ptr<Expression>> expressions;
 		for (auto &column_id : column_ids) {
 			if (column_id.IsRowIdColumn()) {
-				types.emplace_back(LogicalType::ROW_TYPE);
-				expressions.push_back(make_uniq<BoundConstantExpression>(Value::BIGINT(0)));
```

Comment:
Maybe this can just push a `NULL` value (`Value(LogicalType::ROW_TYPE)`)? That way we don't need special code for all these types.